lua/modules/configs/ui/edge.lua

```diff
@@ -4,5 +4,5 @@ return function()
 	vim.g.edge_disable_italic_comment = 1
 	vim.g.edge_show_eob = 1
 	vim.g.edge_better_performance = 1
-	vim.g.edge_transparent_background = 1
+	vim.g.edge_transparent_background = require("core.settings").transparent_background
```

Comment:
```suggestion
	vim.g.edge_transparent_background = require("core.settings").transparent_background and 2 or 0
```

[sainnhe/edge/doc/edge.txt#L252-L265](https://github.com/sainnhe/edge/blob/358cb6688ac577470a4eafcb53bdd63899dfc937/doc/edge.txt#L252-L265)